src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -111,6 +111,24 @@ describe('Test suite for dictionary actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should handle fetch dictionaries network error', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.reject({
+        status: 599,
+      });
+    });
+    const expectedActions = [
+      { payload: true, type: '[ui] toggle spinner' },
+      { payload: false, type: '[ui] toggle spinner' },
```

Comment:
How do these two actions work together at the same time?

Reply:
Method `fetchDictionaries` and `fetchDictionaryConcepts` and others in `dictionaryActionCreators.js` file dispatch both actions.